xbmc/utils/ColorUtils.h

```diff
@@ -19,6 +19,7 @@ namespace COLOR
 {
 
 typedef uint32_t Color;
+typedef uint32_t ColorSelect;
```

Comment:
what is the difference between `Color` and `ColorSelect`?

Reply:
There is currently a mixture about two ways to get the color.
One with `ARGB` format and the other with normal integer number to get color within list, think can be better to declare as `unsigned int` instead of `uint32_t`.

Reply:
but `ColorSelect` type is passed to `character_t` struct this last one use `Color` type argument and then the value is stored/converted again as `ColorSelect` type. In other places the `character_t` value is passed with `Color` type value, all this seems a bit inconsistent and mixed.
Then `ColorSelect` is only a index number for a list, that has nothing to do with a color, why not call it something like `ColorIndex` ?

Reply:
I updated it and now uses `typedef unsigned int ColorIndex;` (really better with index).
I also used `constexpr ColorIndex INDEX_DEFAULT = 0;`, fits better than `UTILS::COLOR::NONE`.

Reply:
now is much better i like it
just remember to change `UTILS::COLOR::Color` to `UTILS::COLOR::ColorIndex` in to `character_t` constructors